### DISCUSSION TITLE
Remove now plying notification

### DISCUSSION THREAD

**Discussion by AKA-Akhil** [end-4/dots-hyprland#552] [Q&A] [answered]
this now playing notification is staying on the screen forever interrupting the work is there a way so that i disappears after a few seconds or minimizes it into an icon or remove it completely
![image](https://github.com/end-4/dots-hyprland/assets/145592032/09d9c18d-b32d-4d47-8c6d-1a6d9e591237)

**Comment by end-4** [comment, marked as answer]
either press that music title on the bar or press super+m
